### DISCUSSION TITLE
Cannot upgrade to Windows 11 24H2

### DISCUSSION THREAD

**Discussion by tolegedeeu** [utmapp/UTM#6718] [Windows]
Running the Windows 11 24H2 setup on an UTM vm using Windows 11 23H2, an issue is reported: processor not compatible (popcnt). I run the upgrade the same way in a VirtualBox machine without any problems.

UTM is installed on the latest Intel MacBook Pro using macOS 14.7.
UTM hardware is: x86_64, Q35 + ICH9, 2009

**Comment by canoben** [comment]
I'm having the same problem, and an addtional one:
<img width="541" alt="utm" src="https://github.com/user-attachments/assets/0147232d-6f07-43c0-8fe5-86346d741d6e" />
(latest UTM 4.6.4)

**Comment by acdiaznavas** [comment]
I have the same problem when trying to update to version 24H2:
![Captura de pantalla 2025-04-15 a las 23 17 05](https://github.com/user-attachments/assets/201f9eb4-4e85-4e9f-9578-1ea1387773dc)

UTM 4.6.4 on an Intel MacBook Pro using macOS 15.4
UTM hardware is: x86_64, Q35 + ICH9, 2009 (pc-q35-9.1), Skylake-Client.

**Comment by mberneis** [comment]
Same Problem 
<img width="463" alt="image" src="https://github.com/user-attachments/assets/b198016e-22f3-4707-9d34-30eb9d3d257d" />

**Comment by acdiaznavas** [comment]
The problem persists.
Furthermore, a recent update (repair version) for Windows 11 23H2 was released. When it reaches 8% download speed, it starts a process to check if the processor is compatible and now shows that the processor is incompatible with the default processor configuration. I fear that Windows 11 will no longer be able to be updated under these conditions.
It's been a long time since this discussion thread was opened, and it's strange that no expert has commented on it yet...

**Comment by acdiaznavas** [comment]
Currently updated to UTM version 4.6.5 (108) from the App Store and the problem still persists, is there any expert who can address this serious issue?

**Comment by ideologysec** [comment]
Please specify the exact model of your Intel processor and/or Macbook.

Please provide a link or a copy to download your UTM config file (and ideally your VM if you're comfortable sharing) for troubleshooting purposes.
